### PR TITLE
GVHP-1-Api Portal: Updating the yaml file

### PR DESCRIPTION
**Pull request by shivanispande**

### CODE REVIEW

File: reference/1.0.0/issued-device-admin-1.0.0.yaml [hunk 1]
```diff
@@ -90,9 +90,9 @@ paths:
       tags:
       - Helix Core
       summary: This method is used to create new card
-      description: Add a new card API allows the user to add a new card to the platform.
-        The new card will be linked to the Person ID and Account ID provided in the
-        request message. It is not intended to be used to replace an existing card.
+      description: ' This API allows the user to add a new card to the platform. The
```
Comment (tejomurthy) [inline reference/1.0.0/issued-device-admin-1.0.0.yaml:93]: Please remove additional space before This API

File: reference/1.0.0/issued-device-admin-1.0.0.yaml [hunk 2]
```diff
@@ -286,8 +286,8 @@ paths:
       tags:
       - Helix Core
       summary: This method is used to activate a card
-      description: "Activating a card API allows the user to activate a card, when\
-        \ a card requires activation prior to first use."
+      description: "This API allows the user to activate a card, when a card requires\
```
Comment (tejomurthy) [inline reference/1.0.0/issued-device-admin-1.0.0.yaml:289]: This description is not very good. This API allows activation of a card. This should be called if the card requires activation. Check with BAs to see if change like this is appropriate

File: reference/1.0.0/issued-device-admin-1.0.0.yaml [hunk 3]
```diff
@@ -631,8 +613,8 @@ paths:
       - Helix Core
       summary: This method is used to return the issued device details for the given
         FEPID
-      description: Inquire on card details API allows the user to retrieve details
-        of a specific card in the platform.
+      description: This API allows the user to retrieve details of a specific card
+        in the platform.
```
Comment (tejomurthy) [inline reference/1.0.0/issued-device-admin-1.0.0.yaml:617]: Not sure of this description either.

**Review by tejomurthy — APPROVED** [review]

File: reference/1.0.0/issued-device-admin-1.0.0.yaml (hunk 2 above)
Comment (tejo-fiserv) [reply reference/1.0.0/issued-device-admin-1.0.0.yaml:289]: Ignore this comment for now.

**Review by tejo-fiserv — APPROVED** [review]

**Pull request merged**